src/main/scala/akka/persistence/cassandra/query/CassandraReadJournalConfig.scala

```diff
@@ -0,0 +1,23 @@
+package akka.persistence.cassandra.query
+
+import scala.concurrent.duration._
+import com.typesafe.config.Config
+import com.datastax.driver.core.ConsistencyLevel
+import akka.persistence.cassandra.journal.CassandraJournalConfig
+
+private[query] class CassandraReadJournalConfig(config: Config, writePluginConfig: CassandraJournalConfig) {
```

Comment:
Nothing more than needed have to be configured, since default values from reference.conf work fine.
Duplicating properties is not nice, and a common config section will not work because we must support multiple instances with different config.

To me, the write plugin "cassandra-journal" is the mandatory and most important, and therefore it makes sense to me that the common things are defined in that section.

That said, I don't have a strong opinion about this and you are welcome to refactor it in another PR.